src/flows/pipes/Visualization/threshold.ts

```diff
@@ -18,61 +18,54 @@ export interface ErrorThreshold extends Threshold {
   fieldType: string
 }
 
+export const lambdaPrefix = '(r) =>'
+
+export const EQUALITY_THRESHOLD_TYPES = {
+  equal: {
+    name: 'equal to',
+    format: ThresholdFormat.Value,
+    condition: data => `${lambdaPrefix} (r["${data.field}"] == ${data.value})`,
+  },
+  'not-equal': {
+    name: 'not equal to',
+    format: ThresholdFormat.Value,
+    condition: data => `${lambdaPrefix} (r["${data.field}"] != ${data.value})`,
+  },
+}
+
 export const COMMON_THRESHOLD_TYPES = {
+  ...EQUALITY_THRESHOLD_TYPES,
```

Comment:
We should have a single point of definition for `equal` and `not-equal`.
Rather than having 2 points of definition.